changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="utf-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Marcin Mogiela" id="1">
+    <tagDatabase tag="v1.1.0" />
+  </changeSet>
+  <changeSet author="Marcin Mogiela" id="2">
+    <createTable tableName="eligibility">
+      <column name="sbi" type="NUMBER(9)">
+        <constraints nullable="true" unique="true" uniqueConstraintName="elg_sbi_ukey" />
+      </column>
+      <column name="crn" type="NUMBER(10)">
+        <constraints nullable="true" unique="true" uniqueConstraintName="elg_crn_ukey" />
+      </column>
+      <column name="customer_name" type="VARCHAR2(1283)">
+        <constraints nullable="true" />
+      </column>
+      <column name="business_name" type="VARCHAR2(160)">
+        <constraints nullable="true" />
+      </column>
+      <column name="business_email" type="VARCHAR2(254)">
+        <constraints nullable="true" unique="true" uniqueConstraintName="elg_business_email_ukey" />
+      </column>
+      <column name="business_address" type="VARCHAR2(4000)">
+        <constraints nullable="true" />
+      </column>
```

Comment:
Maybe we can have a last updated timestamp column as well - might be useful 🤷‍♂️

Reply:
done + added an index to it

<img width="976" alt="image" src="https://user-images.githubusercontent.com/118533775/206135622-338fb770-40cf-47bd-9bf4-ee995a0e7926.png">